src/pages/fr/core-concepts/project-structure.md

```diff
@@ -0,0 +1,105 @@
+
+Voici comment un projet Astro est organisé, et les quelques fichiers que vous trouverez dans votre nouveau projet.
+
+## Répertoires et fichiers
+
+Astro influence une façon d'organiser de dossier pour votre projet. Chaque projet Astro doit inclure les répertoires et fichiers suivants :
+
+- `src/*` - Le code source de votre projet (composants, pages, styles, etc.)
+- `public/*` - Tout ce qu'il n'est pas du code, fichiers non-traités (polices d'écritures, icones, etc.)
+- `package.json` - Le manifeste de votre projet.
+- `astro.config.mjs` - Un fichier de configuration d'Astro (optionel).
+
+### Exemple de structure de projet
+
+Un projet Astro assez commun peut ressembler à ça :
+
+```
+├── src/
+│   ├── components/
+│   │   ├── Header.astro
+│   │   └-─ Button.jsx
+│   ├── layouts/
+│   │   └-─ PostLayout.astro
+│   └── pages/
+│   │   ├── posts/
+│   │   │   ├── post1.md
+│   │   │   ├── post2.md
+│   │   │   └── post3.md
+│   │   └── index.astro
+│   └── styles/
+│       └-─ global.css
+├── public/
+│   ├── robots.txt
+│   ├── favicon.svg
+│   └-─ social-image.png
+├── astro.config.mjs
+└── package.json
+
+```
+
+### `src/`
+
+Le dossier src est où se trouve le code source de votre projet. Il comprend :
+
+- [Des composants Pages](/fr/core-concepts/astro-pages/)
+- [Des composants Layouts](/fr/core-concepts/layouts/)
+- [Des composants Astro](/fr/core-concepts/astro-components/)
+- [Des composants Front-end (React, etc.)](/fr/core-concepts/framework-components/)
+- [Des fichiers de style (CSS, Sass)](/fr/guides/styling/)
+- [Du Markdown](/fr/guides/markdown-content/)
+
+Astro traite, optimise et regroupe les fichiers `src/` pour créer le site web final qui est délivré au navigateur. Contrairement au répertoire statique `public/`, les fichiers `src/` sont traités et assemblés pour vous par Astro.
+
+Quelques fichiers (comme les composants Astro) ne sont pas envoyés au navigateur tel quel, mais plutôt en temps qu'HTML statique. D'autres fichiers (comme les CSS) sont envoyés au navigateur, mais peuvent être optimisés ou regroupés avec d'autres fichiers CSS pour améliorer les performances.
+
+### `src/components`
+
+**Les composants** sont généralement du code réutilisables pour vos pages HTML. Ils peuvent être des [composants Astro](/fr/core-concepts/astro-components/), ou des [composants frontend](/fr/core-concepts/framework-components/) comme React ou Vue. Il est commun de grouper et d'organiser tous les composants de votre projet dans ce dossier.
+
+C'est une convention commune dans les projets Astro, mais c'est pas obligatoire. Organisez vos composants comme vous le voulez, si vous le souhaitez !
+
+### `src/layouts`
+
+[Les composants Layouts](/fr/core-concepts/layouts/) sont des types de composants particuliers qui s'appliquent à des pages. Ils sont généralement utilisés par les [Pages Astro](/fr/core-concepts/astro-pages/) et les [Pages Markdown](/fr/guides/markdown-content/) pour définir leur mise en page.
```

Comment:
"...mise-en-page" with hyphens? (I'm very uncertain about this one.)

Reply:
According to the dictionary, it don't takes hyphen i think